src/app/pcasts/dao/notifications_dao.py

```diff
@@ -0,0 +1,55 @@
+import json
+from . import *
+from app.pcasts.dao import series_dao, episodes_dao
+
+def new_episode_notifications(user_id, max_episodes, offset):
+  subscriptions = get_subscribed_series_for_new_episodes(user_id)
+  series_ids = [s.series_id for s in subscriptions]
+  episodes = episodes_dao.get_episodes_by_many_series(user_id, series_ids,  \
+      offset, max_episodes)
+  # Populate series data
+  series = series_dao.get_multiple_series(series_ids, user_id)
+  series_map = {s.id: s for s in series}
+  for e in episodes:
+    e.series = series_map[e.series_id]
+    e.unread_notifcation = is_unread_notification(user_id, e.id)
```

Comment:
do we want to batch these?

Reply:
This is probably the correct move and is trivial to implement but for now i just want to release this. I have added a todo in slack that i can take care of after the meeting today.